README.md

```diff
@@ -4,7 +4,69 @@ bookend is a classifier for text data. We are building a machine learning model
 
 This project is part of The Erdos Institute's  [data science boot camp](https://www.erdosinstitute.org/code). Data sources include [Project Gutenberg](https://www.gutenberg.org/).
 
-## Getting started
+## Table of contents
+
+- [About the project](#about-the-project)
+  - [Methodology](#methodology)
+- [Results](#results)
+- [Applications](#applications)
+- [Using the code](#using-the-code)
+- [References](#references)
+
+## About the project
+
+The goal of this project is to use the body of a text to classify it. To start, we are focusing on predicting the author of a text from the text itself. If we give our algorithm 
+> It was the best of times, it was the worst of times...
+
+will it know Charles Dickens wrote it?
+
+### Methodology
+
+We build many features out of the text, and use an ensemble machine learning model to predict the author.
+
+In the future we will expand to allow any-size snippets of text and additional authors, but for now we have focused on identifying 75-sentence chunks of text. We include the following authors, chosen as the top authors on Project Gutenberg who wrote their works in English (We exclude Shakespeare, as his works are largely plays rather than novels or short stories. We additionally add JK Rowling, who famously wrote the Harry Potter series as well as crime fiction novels under the pseudonym Robert Galbraith):
```

Comment:
In addition to Shakespeare we are also excluding Poe, correct? Also, I don't see Twain on the list below, but I'm still not sure if we're actually using him.

Reply:
I added a bit about Poe. I think our plan is to add Twain back in.